examples/gno.land/r/demo/pagination/pagination_test.gno

```diff
@@ -0,0 +1,61 @@
+package pagination
+
+import (
+	"std"
+	"testing"
+
+	"gno.land/p/demo/testutils"
+	"gno.land/p/demo/ufmt"
+)
+
+func TestRenderHome(t *testing.T) {
+	// Sign up 50 users
+	for i := 0; i < 50; i++ {
+		addrName := ufmt.Sprintf("test%d", i)
+		caller := testutils.TestAddress(addrName)
+		std.TestSetOrigCaller(caller)
+		SignUp()
+	}
+
+	render := Render("")
```

Comment:
It would be good to test the first page and a subsequent page; basically something where a non-empty string is passed to `Render`. Some edge and non-typical cases would be good too.

Reply:
Added some better tests: [8ca5956](https://github.com/gnolang/gno/pull/1949/commits/8ca5956cfffc06dbe0f6cb6c4b77bc86220d48e6)

WDYT?